Only in maven-1.0.2-src/bootstrap/install-phase1/lib: maven.jar
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: clean
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: jar
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: java
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: license
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: plugin
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: project.xml
Only in maven-1.0.2-src/bootstrap/install-phase1/plugins: test
Files maven-1.0.2-src/bootstrap/maven.jar and maven-1.0.2-src-patched/bootstrap/maven.jar differ
diff -ru maven-1.0.2-src/build-bootstrap.xml maven-1.0.2-src-patched/build-bootstrap.xml
--- maven-1.0.2-src/build-bootstrap.xml	2005-09-05 14:23:59.000000000 -0400
+++ maven-1.0.2-src-patched/build-bootstrap.xml	2005-12-30 17:24:16.000000000 -0500
@@ -38,9 +38,7 @@
    <property name="clo-jar" value="commons-logging/jars/commons-logging-1.0.3.jar" />
    <property name="chc-jar" value="commons-httpclient/jars/commons-httpclient-2.0.jar" />
    <path id="bootstrap.libs">
-     <pathelement location="${maven.repo.local}/${cla-jar}" />
-     <pathelement location="${maven.repo.local}/${clo-jar}" />
-     <pathelement location="${maven.repo.local}/${chc-jar}" />
+     <fileset dir="lib" includes="*.jar"/>
    </path>
 <echo>
 +----------------------------------------------------------
@@ -96,17 +94,19 @@
 
     <delete dir="${maven.bootstrap.bin.dir}"/>
     <mkdir dir="${maven.bootstrap.bin.dir}"/>
+	<!--
     <bootstrap
       projectDescriptor="project.xml"
       baseUrl="${maven.get.jars.baseUrl}/"
       mavenRepoLocal="${maven.repo.local}"
-      online="${maven.bootstrap.online}"
+      online="false"
       proxyHost="${maven.proxy.host}"
       proxyUserName="${maven.proxy.username}"
       proxyPassword="${maven.proxy.password}"
       proxyPort="${maven.proxy.port}"
       copyScriptDir="${maven.bootstrap.bin.dir}"
     />
+	-->
 
 <echo>
 |
@@ -122,7 +122,7 @@
       debug="on"
       deprecation="off"
       optimize="off">
-      <classpath refid="bootstrap-classpath"/>
+      <classpath refid="bootstrap.libs"/>
       <src>
         <path location="src/java"/>
       </src>
@@ -155,29 +155,36 @@
       </fileset>
     </jar>
 
+<!--
 <echo>
 |
 | CREATING PHASE 1 INSTALL...
 |
 </echo>
+-->
 
     <!-- Create the install directory -->
 
+	<!--
     <delete dir="${maven.bootstrap.install.dir}"/>
     <mkdir dir="${maven.bootstrap.install.dir}"/>
     <mkdir dir="${maven.bootstrap.install.dir}/bin"/>
     <mkdir dir="${maven.bootstrap.install.dir}/lib"/>
     <mkdir dir="${maven.bootstrap.install.dir}/plugins"/>
+	-->
 
     <!-- Copy Maven's scripts into the install bin directory -->
 
+	<!--
     <copy todir="${maven.bootstrap.install.dir}/bin">
       <fileset dir="src/bin"/>
       <fileset dir="${maven.bootstrap.bin.dir}"/>
     </copy>
+	-->
 
     <!-- Copy Maven's dependencies into install lib directory -->
 
+	<!--
     <copy
       todir="${maven.bootstrap.install.dir}/lib"
       flatten="yes">
@@ -185,8 +192,10 @@
         <patternset refid="bootstrap-patternset"/>
       </fileset>
     </copy>
+	-->
 
     <!-- make the endorsed dir -->
+	<!--
     <mkdir dir="${maven.bootstrap.install.dir}/lib/endorsed" />
     <move todir="${maven.bootstrap.install.dir}/lib/endorsed">
       <fileset dir="${maven.bootstrap.install.dir}/lib">
@@ -194,25 +203,31 @@
         <include name="xerces-*.jar" />
       </fileset>
     </move>
+	-->
 
     <!-- Don't duplicate endorsed files -->
+	<!--
     <delete>
       <fileset dir="${maven.bootstrap.install.dir}/lib">
         <include name="xml-apis-*.jar" />
         <include name="xerces-*.jar" />
       </fileset>
     </delete>
+	-->
 
     <!-- Install maven in the lib directory -->
 
+	<!--
     <copy
       file="${maven.bootstrap.dir}/maven.jar"
       todir="${maven.bootstrap.install.dir}/lib"
       overwrite="true"
     />
+	-->
 
     <!-- Install maven in the local repository for the plugin builds -->
 
+	<!--
     <copy
       file="${maven.bootstrap.dir}/maven.jar"
       todir="${maven.repo.local}/maven/jars"
@@ -233,8 +248,10 @@
     <copy todir="${maven.home}">
       <fileset dir="${maven.bootstrap.install.dir}" />
     </copy>
+	-->
 
     <!-- Now we have a functional maven install -->
+	<!--
     <chmod file="${maven.home}/bin/maven" perm="+x"/>
 
     <condition property="maven.script" value="maven.bat">
@@ -273,6 +290,7 @@
     <copy todir="${maven.bootstrap.install.dir}/plugins">
       <fileset dir="${maven.home}/plugins" />
     </copy>
+	-->
 
 <!-- Don't test the plugins, it breaks too often at the fault of rogue plugins
     <echo>
@@ -293,6 +311,7 @@
     </exec>
 -->
 
+	<!--
     <echo>
 |
 | BUILDING MAVEN WITH ITSELF...
@@ -309,9 +328,11 @@
     </exec>
 
     <antcall target="clean-maven-home"/>
+	-->
 
     <!-- Use the newly created maven.jar -->
 
+	<!--
     <copy
       file="${basedir}/target/maven.jar"
       todir="${maven.repo.local}/maven/jars"
@@ -341,6 +362,7 @@
     </copy>
 
     <chmod file="${maven.home}/bin/maven" perm="+x"/>
+	-->
   </target>
 
   <!-- ===================================================================== -->
@@ -357,9 +379,11 @@
     <mkdir dir="${maven.repo.local}/commons-lang/jars"/>
     <mkdir dir="${maven.repo.local}/commons-logging/jars"/>
     <mkdir dir="${maven.repo.local}/commons-httpclient/jars"/>
+	<!--
     <get ignoreerrors="true" usetimestamp="true" dest="${maven.repo.local}/${cla-jar}" src="${maven.get.jars.baseUrl}/${cla-jar}" />
     <get ignoreerrors="true" usetimestamp="true" dest="${maven.repo.local}/${clo-jar}" src="${maven.get.jars.baseUrl}/${clo-jar}" />
     <get ignoreerrors="true" usetimestamp="true" dest="${maven.repo.local}/${chc-jar}" src="${maven.get.jars.baseUrl}/${chc-jar}" />
+	-->
 
     <javac
       destdir="${maven.bootstrap.classes}"
Only in maven-1.0.2-src-patched: .build-bootstrap.xml.swo
Only in maven-1.0.2-src-patched: .build-bootstrap.xml.swp
Only in maven-1.0.2-src-patched: hs_err_pid16202.log
Only in maven-1.0.2-src: maven.log
Only in maven-1.0.2-src-patched: .maven.xml.swp
Only in maven-1.0.2-src-patched: .README.txt.swp
Only in maven-1.0.2-src/tmp/bin: forehead.conf
Only in maven-1.0.2-src/tmp/bin: maven
Only in maven-1.0.2-src/tmp/bin: maven.bat
Only in maven-1.0.2-src/tmp/lib: ant-1.5.3-1.jar
Only in maven-1.0.2-src/tmp/lib: ant-optional-1.5.3-1.jar
Only in maven-1.0.2-src/tmp/lib: commons-beanutils-1.6.1.jar
Only in maven-1.0.2-src/tmp/lib: commons-betwixt-1.0-beta-1.20030111.103454.jar
Only in maven-1.0.2-src/tmp/lib: commons-cli-1.0-beta-2.jar
Only in maven-1.0.2-src/tmp/lib: commons-collections-2.1.jar
Only in maven-1.0.2-src/tmp/lib: commons-digester-1.4.1.jar
Only in maven-1.0.2-src/tmp/lib: commons-grant-1.0-beta-4.jar
Only in maven-1.0.2-src/tmp/lib: commons-graph-0.8.1.jar
Only in maven-1.0.2-src/tmp/lib: commons-httpclient-2.0.jar
Only in maven-1.0.2-src/tmp/lib: commons-io-20030203.000550.jar
Only in maven-1.0.2-src/tmp/lib: commons-jelly-20030902.160215.jar
Only in maven-1.0.2-src/tmp/lib: commons-jelly-tags-ant-1.0.jar
Only in maven-1.0.2-src/tmp/lib: commons-jelly-tags-define-20030211.142932.jar
Only in maven-1.0.2-src/tmp/lib: commons-jelly-tags-util-20030211.141939.jar
Only in maven-1.0.2-src/tmp/lib: commons-jelly-tags-xml-20040613.030723.jar
Only in maven-1.0.2-src/tmp/lib: commons-jexl-1.0-beta-1.jar
Only in maven-1.0.2-src/tmp/lib: commons-lang-2.0.jar
Only in maven-1.0.2-src/tmp/lib: commons-logging-1.0.3.jar
Only in maven-1.0.2-src/tmp/lib: dom4j-1.4-dev-8.jar
Only in maven-1.0.2-src/tmp/lib: endorsed
Only in maven-1.0.2-src/tmp/lib: forehead-1.0-beta-5.jar
Only in maven-1.0.2-src/tmp/lib: log4j-1.2.8.jar
Only in maven-1.0.2-src/tmp/lib: maven.jar
Only in maven-1.0.2-src/tmp/lib: maven-jelly-tags-1.0.1.jar
Only in maven-1.0.2-src/tmp/lib: plexus-0.6.jar
Only in maven-1.0.2-src/tmp/lib: werkz-20040426.222000.jar
Only in maven-1.0.2-src/tmp/lib: which-1.0.jar
Only in maven-1.0.2-src/tmp/plugins: clean
Only in maven-1.0.2-src/tmp/plugins: jar
Only in maven-1.0.2-src/tmp/plugins: java
Only in maven-1.0.2-src/tmp/plugins: license
Only in maven-1.0.2-src/tmp/plugins: maven-abbot-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-announcement-plugin-1.4-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-antlr-plugin-1.2.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-ant-plugin-1.10-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-castor-plugin-1.2.jar
Only in maven-1.0.2-src/tmp/plugins: maven-changelog-plugin-1.8.2.jar
Only in maven-1.0.2-src/tmp/plugins: maven-checkstyle-plugin-3.0-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-clean-plugin-1.4-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-clover-plugin-1.10.jar
Only in maven-1.0.2-src/tmp/plugins: maven-console-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-dashboard-plugin-1.9-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-developer-activity-plugin-1.5.2.jar
Only in maven-1.0.2-src/tmp/plugins: maven-dist-plugin-1.7-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-ear-plugin-1.7.jar
Only in maven-1.0.2-src/tmp/plugins: maven-eclipse-plugin-1.10-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-ejb-plugin-1.7.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-faq-plugin-1.5-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-file-activity-plugin-1.5.2.jar
Only in maven-1.0.2-src/tmp/plugins: maven-genapp-plugin-2.3-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-gump-plugin-2.0.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-hibernate-plugin-1.3.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-html2xdoc-plugin-1.3.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-idea-plugin-1.6.jar
Only in maven-1.0.2-src/tmp/plugins: maven-j2ee-plugin-1.5.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jalopy-plugin-1.3.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jar-plugin-1.7.jar
Only in maven-1.0.2-src/tmp/plugins: maven-javacc-plugin-1.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-java-plugin-1.5.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jboss-plugin-1.6-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jcoverage-plugin-1.0.10-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jdiff-plugin-1.5-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jellydoc-plugin-1.3.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jetty-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jira-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jnlp-plugin-1.5-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-junit-report-plugin-1.5.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-jxr-plugin-1.4.3.jar
Only in maven-1.0.2-src/tmp/plugins: maven-license-plugin-1.2.jar
Only in maven-1.0.2-src/tmp/plugins: maven-multichanges-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-multiproject-plugin-1.4.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-native-plugin-1.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-nsis-plugin-2.0-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-pdf-plugin-2.4-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-plugin-plugin-1.6.jar
Only in maven-1.0.2-src/tmp/plugins: maven-pmd-plugin-1.7.jar
Only in maven-1.0.2-src/tmp/plugins: maven-pom-plugin-1.4.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-rar-plugin-1.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-release-plugin-1.4.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-repository-plugin-1.2.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-scm-plugin-1.5.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-simian-plugin-1.5.jar
Only in maven-1.0.2-src/tmp/plugins: maven-site-plugin-1.6.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-tasklist-plugin-2.4.jar
Only in maven-1.0.2-src/tmp/plugins: maven-test-plugin-1.7.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-uberjar-plugin-1.2.1-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: maven-war-plugin-1.6.1.jar
Only in maven-1.0.2-src/tmp/plugins: maven-xdoc-plugin-1.9.2-SNAPSHOT.jar
Only in maven-1.0.2-src/tmp/plugins: plugin
Only in maven-1.0.2-src/tmp/plugins: project.xml
Only in maven-1.0.2-src/tmp/plugins: test
